website/docs/r/postgresql_flexible_server.html.markdown

```diff
@@ -116,6 +116,12 @@ The following arguments are supported:
 
 ~> **Note:** There will be a breaking change from upstream service at 15th July 2021, the `private_dns_zone_id` will be required when setting a `delegated_subnet_id`. For existing flexible servers who don't want to be recreated, you need to provide the `private_dns_zone_id` to the service team to manually migrate to the specified private DNS zone. The `azurerm_private_dns_zone` should end with suffix `.postgres.database.azure.com`.
 
+* `public_network_access_enabled` - (Optional) Is public network access enabled?
```

Comment:
```suggestion
* `public_network_access_enabled` - (Optional) Specifies whether this PostgreSQL Flexible Server is publicly accessible. Defaults to `true`.
```

Reply:
As I above mentioned, we can't simply hard-code the default value for public_network_access_enabled. So we can't add "Defaults to `true`." here.

Reply:
Updated